include/wx/hwinfo.h

```diff
@@ -0,0 +1,40 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/hwinfo.h
+// Purpose:     declaration of the wxSystemHardwareInfo class
+// Author:      Blake Madden
+// Created:     07.09.2024
```

Comment:
According to the the [Coding Guidelines](https://wxwidgets.org/develop/coding-guidelines/#wxwin_header), ISO format should be used.
```suggestion
// Created:     2024-07-09
```

The headers in other files are also not in accordance with the guidelines